data/transactions/logic/eval_test.go

```diff
@@ -5061,116 +5092,6 @@ func TestBytesConversions(t *testing.T) {
 	testAccepts(t, "byte 0x0011; byte 0x10; b+; btoi; int 0x21; ==", 4)
 }
 
-func TestLog(t *testing.T) {
-	partitiontest.PartitionTest(t)
```

Comment:
relocated to evalStateful_test

Reply:
FWIW, the reason I did this churn was to discourage the use of `makeSampleEnv()` in eval_test.go. It creates an eval params for evaluating apps, which these tests do indeed need, but I noticed several tests in `eval_test.go` that _don't_ need it trying to use them and (now) failing because they were only testing logicsigs.